adk/ADK.py

```diff
@@ -28,7 +25,11 @@ def __init__(self, apply_func, load_func=None):
         else:
             self.load_func = None
         if len(apply_args) > 2 or len(apply_args) == 0:
-            raise Exception("apply function may have between 1 and 2 parameters, not {}".format(len(apply_args)))
+            raise Exception(
+                "apply function may have between 1 and 2 parameters, not {}".format(
```

Comment:
Not sure if I agree with the line breaks; this is less agregious but the one below is definitely less legible

Reply:
That's my code formatter `black`'s opinion but I don't object to your argument either hehe